stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -151,7 +152,7 @@ pub fn boot_nakamoto<'a>(
         .map(|test_stacker| {
             (
                 PrincipalData::from(key_to_stacks_addr(&test_stacker.stacker_private_key)),
-                u64::try_from(test_stacker.amount).expect("Stacking amount too large"),
+                u64::try_from(test_stacker.amount + 10000).expect("Stacking amount too large"),
```

Comment:
I think we should do this in a nicer way. We had to explicitly add this for our reasons but it is not clear to anyone using the test helpers.

I'm not sure but I think you can add the test stackers to initial balances. So from your test you will pass an initial balance for Stacker 1, and revert the change on this line.

If that does not work because the initial balances list does not allow duplicates, then we have to modify the code here to sum duplicates.

Reply:
Let's do sponsored txs.

Reply:
Apparently, we don't need it

Reply:
Don't need what?